modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/index.js

```diff
@@ -387,7 +387,11 @@ const InviteTeamMembersPage = ({
 												  )
 										}
 										  ${i18n.sub('available-x-of-x', [
-												availableAdminsRoles,
+												`${
```

Comment:
Hi @amosfong thanks for the help! In this case you don't need to use string template, you can just use:
```suggestion
												availableAdminsRoles < 0 ? 0 : availableAdminsRoles
```